DataDog/documentation
# [WEB-885] - integration logos

### What does this PR do?

- updates committed integrations to have integration id frontmatter param
- additional logic to grab integration id and add as frontmatter params during processing repos
- adds `integrations-logo.html` partial as a consolidated way to get a logo
- update reference to pull logo from partial
- adds breadcrumb to detail integration pages so you can traverse back to integrations page

### Motivation

https://datadoghq.atlassian.net/browse/WEB-885

### Preview

**Testing**
- Check integration tiles look good at different sizes
- Check integration count/tiles matches production or makes sense if differs
- Check integration detail page breadcrumb at the top of page looks and works correctly at different sizes
- Check security monitoring rules icons are correct and have no display issues at various resolutions
- Check a rule detail page shows the correct integration image

**Urls**
- https://docs-staging.datadoghq.com/david.jones/integrations-logos/integrations/
- https://docs-staging.datadoghq.com/david.jones/integrations-logos/security_monitoring/default_rules/

### Additional Notes


---

### Reviewer checklist
